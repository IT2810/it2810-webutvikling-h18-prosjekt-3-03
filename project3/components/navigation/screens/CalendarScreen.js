import React from 'react';
import {View} from 'react-native';
import CalendarComponent from '../../calendar/CalendarComponent';

export default class CalendarScreen extends React.Component {
    static navigationOptions = {
        title: 'Calendar',
    };

    render() {
        return (
            <View style={{flex: 1, alignItems: 'center', justifyContent: 'center'}}>
                <CalendarComponent/>
            </View>
        );
    }
}
