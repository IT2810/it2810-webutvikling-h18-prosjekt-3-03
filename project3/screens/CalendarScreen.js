import React from 'react';
import { View, Text } from 'react-native';


export default class CalendarScreen extends React.Component {
    static navigationOptions = {
        title: 'Calendar',
    };
    
    render() {
        return(
            <View style={{flex: 1,alignItems: 'center', justifyContent: 'center'}}>
                <Text>Calendar screen</Text>
            </View>
        );
    }

}