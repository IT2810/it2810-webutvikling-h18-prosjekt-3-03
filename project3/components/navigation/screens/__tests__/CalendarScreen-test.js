import React from 'react';
import renderer from 'react-test-renderer';
import CalendarScreen from '../CalendarScreen.js';

it('renders correctly', () => {
    // TODO: test shallowly to avoid having to recreate snapshot if subcomponents change
    const tree = renderer.create(<CalendarScreen/>);
    expect(tree.toJSON().children[0].children.length).toBe(3);
    expect(tree.toJSON().children[0].type).toBe('View');
    expect(tree).toMatchSnapshot();
});
