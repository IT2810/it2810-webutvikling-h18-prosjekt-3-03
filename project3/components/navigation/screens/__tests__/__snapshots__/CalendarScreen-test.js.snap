// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
      }
    }
  >
    <View
      style={
        Object {
          "borderColor": "#eeeeee",
          "borderWidth": 1,
          "flex": 1,
          "justifyContent": "center",
        }
      }
    />
    <View
      style={
        Object {
          "flex": 0.4,
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="  Add something"
        style={
          Object {
            "borderColor": "#eee",
            "borderRadius": 8,
            "borderWidth": 3,
            "height": 40,
            "margin": 5,
          }
        }
        value=""
      />
      <View
        style={
          Array [
            Object {
              "marginLeft": 15,
              "marginRight": 15,
            },
            undefined,
            undefined,
            undefined,
          ]
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={null}
        >
          <View
            pointerEvents="box-only"
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#9E9E9E",
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 19,
                },
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Object {
                  "padding": 12,
                },
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
              ]
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {},
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  Array [
                    Object {
                      "color": "white",
                      "fontSize": 20,
                    },
                    undefined,
                    Object {
                      "fontSize": 17.5,
                    },
                    undefined,
                    undefined,
                    undefined,
                    undefined,
                    undefined,
                  ],
                ]
              }
            >
              Add activity
            </Text>
          </View>
        </View>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "alignSelf": "center",
            "color": "#ffa500",
            "fontSize": 15,
            "padding": 5,
          }
        }
      >
        Selected day: 
        2018-10-19
      </Text>
    </View>
    <View
      style={
        Object {
          "flex": 0.5,
        }
      }
    />
  </View>
</View>
`;
